Makefile

```diff
@@ -473,8 +473,48 @@ rpm:
 
 .PHONY: docker-image
 docker-image: static
-	cp $$GOPATH/bin/skydive contrib/docker/
-	docker build -t ${DOCKER_IMAGE}:${DOCKER_TAG} -f contrib/docker/Dockerfile contrib/docker/
+	cp $$GOPATH/bin/skydive contrib/docker/skydive.$$(uname -m)
```

Comment:
is this target still used ?

Reply:
seems I'll be moving back to using this one on amd64.